fuel/src/test/kotlin/com/github/kittinunf/fuel/BlockingRequestTest.kt

```diff
@@ -152,7 +152,7 @@ class BlockingRequestTest : BaseTestCase() {
 
     @Test
     fun httpGetRequestWithPathStringConvertible() {
-        val (request, response, data) = manager.request(Method.GET, HttpsBin.USER_AGENT).responseString()
+        val (request, response, data) = manager.request(Method.GET, HttpsBin.USER_AGENT.path).responseString()
```

Comment:
Should this still work without the change? I should correctly resolve to use the `PathStringConvertible` version.
`